### ISSUE TITLE
Create release schedule (incl. history)

### ISSUE THREAD

**Issue by JamiesHQ** [jupyter/project-mgt#25]
I created a Google sheet to track releases until we can find a way to do this automatically: http://bit.ly/jupyter-releases

**Comment by willingc** [comment]
@JamiesHQ I think you need to change the permissions on the Google sheet.

**Comment by JamiesHQ** [comment]
@willingc : yes, that was my next step. was not expecting an instantaneous response at this hour! sharing permissions are now enabled and I'll continue working on it this week.

**Comment by willingc** [comment]
We can probably pull some of this from the GitHub API to a .csv file if you are looking for historical stuff.

**Comment by JamiesHQ** [comment]
@willingc : yeah! let's do it! show me the way

**Comment by willingc** [comment]
Also the libraries.io project is a good source for historical info too. https://libraries.io/pypi/jupyterhub

**Comment by minrk** [comment]
@JamiesHQ I made [this script](https://gist.github.com/minrk/196d7e46c4a9aa10325069ab8a84b1f4) to get all tagged releases across our repos. A CSV is attached to the gist, which can be imported into the Google spreadsheet.

**Comment by JamiesHQ** [comment]
@minrk wow!! this is awesome!!! thank you thank you thank you!!!

I've added the link to your gist to the PjM reports Wiki and hope to explore it more in the near future (after grant report is submitted): https://github.com/jupyter/project-mgt/wiki/Metrics,-Reports,-and-Feeds 

I will also work with Brian & Cameron to see where we can post this (or at least the latest releases) on our website.

**Comment by JamiesHQ** [comment]
Here's the link to the spreadsheet: https://docs.google.com/spreadsheets/d/16fpjrU2W2YBWUFrFFyY3N5IxNZRykuoZhKkRJE_ulYk/edit#gid=0

looking forward to automating this!

**Comment by JamiesHQ** [comment]
@minrk : could you run the script another time to catch anything new? Fernando would like to add this year's releases to the grant report that we're sending out eod Tuesday and it would be great to show everything through February. I'd like to work to automate it in the coming weeks. Thanks!

**Comment by minrk** [comment]
@JamiesHQ I updated the gist with a fresh run.
